sys/include/net/skald.h

```diff
@@ -67,68 +67,21 @@ extern "C" {
 #endif
 
 /**
- * @brief   Configure advertising channel 37
+ * @brief   Kconfig symbol to store advertising channels as a string.
  *
- * Set CONFIG_ADV_CH_37_DISABLE to disable channel 37
+ * String value stored in CONFIG_SKALD_ADV_CHANNELS is parsed to an ordered
+ * list ( @ref SKALD_ADV_CHAN) in sys/net/ble/skald/Makefile.include.
  */
-#ifdef DOXYGEN
-#define CONFIG_ADV_CH_37_DISABLE
-#endif
-
-/**
- * @brief   Configure advertising channel 38
- *
- * Set CONFIG_ADV_CH_38_DISABLE to disable channel 38
- */
-#ifdef DOXYGEN
-#define CONFIG_ADV_CH_38_DISABLE
-#endif
-
-/**
- * @brief   Configure advertising channel 39
- *
- * Set CONFIG_ADV_CH_39_DISABLE to disable channel 39
- */
-#ifdef DOXYGEN
-#define CONFIG_ADV_CH_39_DISABLE
+#if defined(DOXYGEN)
+#define CONFIG_SKALD_ADV_CHANNELS
```

Comment:
I don't think this needed, it is only used by Kconfig. The user that configures the channels via CFLAGS would use `SKALD_ADV_CHAN` instead. To me adding two knobs creates unnecessary confusion.

Reply:
Where do you think I should document the symbol `CONFIG_SKALD_ADV_CHANNELS` ?

Reply:
It's not used directly in the source code, so I think we need documentation of it in the header file. It's a Kconfig symbol, it already has documentation in the Kconfig file.